### DISCUSSION TITLE
Already Have index.html but show error on github

### DISCUSSION THREAD

**Discussion by makwanaraj** [community/community#23097] [New to GitHub]
<p>I’ve Created my repository and uploaded the project on that in that already added index.html file but when I am trying to make my live on Github URL shows the 404 Error File Not found.</p>
<p>To get better clarity I am sharing SS as well.<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/8/1/8117180987140320a666ff6bf4878b388a9a5608.png" data-download-href="/uploads/short-url/ipZ4tgJO0XX8Vqm5WfP3vwro7jy.png?dl=1" title="2022-06-28 19_21_56-Window" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/25276320/181074841-ab0856dd-3384-4c4b-b84a-eb267867b5c4.png" alt="2022-06-28 19_21_56-Window" data-base62-sha1="ipZ4tgJO0XX8Vqm5WfP3vwro7jy" width="690" height="251" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/8/1/8117180987140320a666ff6bf4878b388a9a5608_2_690x251.png, https://aws1.discourse-cdn.com/github/optimized/3X/8/1/8117180987140320a666ff6bf4878b388a9a5608_2_1035x376.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/8/1/8117180987140320a666ff6bf4878b388a9a5608.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/8/1/8117180987140320a666ff6bf4878b388a9a5608_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">2022-06-28 19_21_56-Window</span><span class="informations">1365×498 18.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div><br>
<div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/a/3/a388ef5480901eb6acc247073b508bdaf6227a81.png" data-download-href="/uploads/short-url/nkHcYCaxBfvJaohTGXuMkSOJIsh.png?dl=1" title="2022-06-28 19_22_27-Window" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/25276320/181074843-4b337749-3a0b-485e-8e69-bfd9472aae10.png" alt="2022-06-28 19_22_27-Window" data-base62-sha1="nkHcYCaxBfvJaohTGXuMkSOJIsh" width="690" height="140" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/a/3/a388ef5480901eb6acc247073b508bdaf6227a81_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">2022-06-28 19_22_27-Window</span><span class="informations">945×192 11.8 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by jfullstackdev** [comment, marked as answer]
<p>did you enable GitHub Pages on the repo settings ? because I cannot see the Environment tab at the repo</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/d/5/d531fc591681285683fb3bb9b8ddabcf0a64df87.png" data-download-href="/uploads/short-url/uq0NixWzIOhepOY9JR36tSVwE7B.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181074849-c22d11fb-2838-48ba-952e-c0135a665c72.png" alt="image" data-base62-sha1="uq0NixWzIOhepOY9JR36tSVwE7B" width="690" height="306" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/d/5/d531fc591681285683fb3bb9b8ddabcf0a64df87_2_690x306.png, https://aws1.discourse-cdn.com/github/optimized/3X/d/5/d531fc591681285683fb3bb9b8ddabcf0a64df87_2_1035x459.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/d/5/d531fc591681285683fb3bb9b8ddabcf0a64df87.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/d/5/d531fc591681285683fb3bb9b8ddabcf0a64df87_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1250×556 25.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by makwanaraj** [comment]
<p>Okay,<br>
now I am able to see my website<br>
Thank you for your help.</p>
